doc/api/net.md

```diff
@@ -551,9 +551,9 @@ changes:
 * `options` {Object} Available options are:
   * `fd` {number} If specified, wrap around an existing socket with
     the given file descriptor, otherwise a new socket will be created.
-  * `allowHalfOpen` {boolean} Indicates whether half-opened TCP connections
-    are allowed. See [`net.createServer()`][] and the [`'end'`][] event
-    for details. **Default:** `false`.
+  * `allowHalfOpen` {boolean} Indicates whether to automatically end the
+    writable side when the readable side ends. See [`net.createServer()`][] and
```

Comment:
Does this make it sound like you should set it to `true` to automatically end the writable side when in fact the opposite is true?

Reply:
1. Indicates whether to *not* automatically end the writable side when the readable side ends
1. Indicates whether to keep the writable side open when the readable side ends.
1. Other suggestions?

Reply:
PTAL I've used the same wording used here https://nodejs.org/api/stream.html#stream_new_stream_duplex_options.